modules/template/src/main/twirl/scaladex/view/project/artifacts.scala.html

```diff
@@ -10,139 +10,43 @@
 @import scaladex.view.Formats
 
 @import java.time.Instant
+
 @(
     env: Env,
     user: Option[UserState],
     project: Project,
-    header: ProjectHeader,
-    artifactName: Artifact.Name,
-    allBinaryVersions: Seq[BinaryVersion],
-    artifactByVersions: SortedMap[(Instant, SemanticVersion), Seq[Artifact]],
-    params: ArtifactsPageParams
+    header: Option[ProjectHeader],
+    groupedArtifacts: Map[SemanticVersion, Seq[Artifact]]
```

Comment:
To get the list of Scala versions of an artifact name, you will need this:

```suggestion
    groupedArtifacts: Map[SemanticVersion, Map[Artifact.Name, Seq[Artifact]]]
```

Reply:
im not getting the ides of why to do this. In the route, I create a Map of [SemanticVersion: Group of Artifacts that have latest version as Semantic Version]

Reply:
For each artifact name, we want to show the list of available Scala versions. So we need several artifacts:
![image](https://github.com/ayushkoli772/scaladex/assets/13123162/e871ec09-172b-4f57-8ef4-f2c73dda7d48)

In the route you should keep all the artifacts with the same name.

Reply:
Got it

Reply:
Got it. I thought we will show them by using platform and binary version for that platform.

Reply:
> I thought we will show them by using platform and binary version for that platform.

Yes you need to use the binary versions: platforms and language versions.